### ISSUE TITLE
label show text "Piauí­" ,Will crash

### ISSUE THREAD

**Issue by hemincan** [CesiumGS/cesium#9334]
Sandcastle example:https://sandcastle.cesium.com/index.html#c=bVHBTgIxEP2VyZ6WBLtBBAIuxGQ9kmhC4qmXoTtgQ7clbXcRDZ/ET+iP2S4QjdJLO2/ee3kzbdBCI2lHFqagaQcFOVlX7KXFUp6Iti6M9ig1WZ507rlugkrhktRcOk8BLlCpJYpN6HG9qrXw0mjAspxHVtqBD64BFqhLgc4rYiUJhZbSCyW6wjkJI+2ll+RY6KatEmBrnIymk0vCAq0PL9R9trKmeqS1JXLpzWjAesO73nA46kJ/zMaD23g63ZNNm3oCZ1MAT29+Ajx5llh/HT+PPDkTD+19iLkOcaifWQKSdJPc+b2i2cXnQVZbYz3UVqWMZZ6qrcKQL1vWYkOeCedOIwLk2W9pXsoGZDm9smkIK3IudFa1Ugv5TjyZ5Vng/5Mqg6XU66eGrMJ9pL32ZvMTyBjLs1BeV3pjwsfZP87f

Browser: Google Chrome 87.0.4280.141

Operating System: windows 10

show on sublime text3
![image](https://user-images.githubusercontent.com/32860032/105146007-6a06e680-5b3a-11eb-825f-a2ca7bd0fab9.png)

copy to Sandcastle :

![image](https://user-images.githubusercontent.com/32860032/105146109-8c006900-5b3a-11eb-80e4-ee7faf6bd628.png)

**Comment by thw0rted** [comment]
Sandcastle hangs the browser tab when I open the demo, but I was able to grab the problem string.  It is `50 69 61 75 ed ad` in hex.  That last character 0xAD is a [soft hyphen](https://en.wikipedia.org/wiki/Soft_hyphen) which is normally not rendered.  It definitely should not be in the label text, but also its inclusion should not cause Cesium to hang the browser.

Actually, I just realized that this already came up in https://github.com/CesiumGS/cesium/issues/4918#issuecomment-452686121 but @mramato said it was fixed.  Maybe they forgot to test for this specific character?

At any rate, IIRC, `Label` never auto-wraps text, which is the only purpose of this character, so simply stripping it from input strings should be the right course of action.
